### ISSUE TITLE
Missing unit tests for `pseudo_kl_divergence` loss function

### ISSUE THREAD

**Issue by kevinchern** [dwavesystems/dwave-pytorch-plugin#56]
It appears in `test_dvae_winci020.py` but is not tested in isolation with determinism; related: https://github.com/dwavesystems/dwave-pytorch-plugin/issues/54
